schema/schemaspec/schemahcl/hcl_test.go

```diff
@@ -109,17 +110,18 @@ show "seinfeld" {
 func ExampleMarshal() {
 	type (
 		Point struct {
-			ID string `spec:",name"`
-			X  int    `spec:"x"`
-			Y  int    `spec:"y"`
+			ID string            `spec:",name"`
+			X  int               `spec:"x"`
+			Y  int               `spec:"y"`
```

Comment:
Please leave the example test as is (it doesn't make sense for a point to have a Z value that is a ref).

Instead add a specific test in https://github.com/ariga/atlas/blob/master/schema/schemaspec/schemahcl/context_test.go